src/libraries/System.Net.Http/tests/StressTests/HttpStress/Directory.Build.props

```diff
@@ -1 +1,17 @@
-<Project/>
+<Project>
+    <PropertyGroup>
+        <PackageRid>linux-x64</PackageRid>
+        <PackageRid Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">win-x64</PackageRid>
+
+        <!-- Stress projects have their own global.json, the directory above that also has it is the repository root. -->
+        <RepositoryRoot>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)../, global.json))/</RepositoryRoot>
+        
+        <TargetingPacksTargetsLocation Condition="'$(TargetingPacksTargetsLocation)' == ''">$(RepositoryRoot)eng/targetingpacks.targets</TargetingPacksTargetsLocation>
```

Comment:
I will not pretend I understand all this targeting stuff, but I'm happy someone from our team does :rofl:

Reply:
I would be also happy if someone explained the terminology properly here. From what I understand, a "targeting pack" allows you to target a specific runtime / tfm in your build, in this case the one you live-build locally. `targetingpacks.targets` somehow makes the globally installed SDK understand it, and allow to build against it, but it needs some ceremony to work.